compile error

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a simple script for reproducing the error.



2. What did you expect to see?

php_swoole.h:918:11: error: 'zend_is_callable_at_frame' was not declared in this scope
     ret = zend_is_callable_at_frame(zcallable, zobject ? Z_OBJ_P(zobject) : NULL, frame, check_flags, &name, fci_cache, error);

3. What did you see instead?



4. What version of Swoole are you using (show your `php --ri swoole`)?



5. What is your machine environment used (show your `uname -a` & `php -v` & `gcc -v`) ?
php8.0rc5
swoole4.5.8

--- a/ext-src/php_swoole.h
+++ b/ext-src/php_swoole.h
@@ -915,7 +915,8 @@ static sw_inline zend_bool sw_zend_is_callable_at_frame(zval *zcallable, zval *z
 #if PHP_VERSION_ID < 80000
     ret = zend_is_callable_ex(zcallable, zobject ? Z_OBJ_P(zobject) : NULL, check_flags, &name, fci_cache, error);
 #else
-    ret = zend_is_callable_at_frame(zcallable, zobject ? Z_OBJ_P(zobject) : NULL, frame, check_flags, &name, fci_cache, error);
+    ret = zend_is_callable_at_frame(zcallable, zobject ? Z_OBJ_P(zobject) : NULL, frame, check_flags, fci_cache, error);
+    name = zend_get_callable_name_ex(zcallable, zobject);
 #endif
     if (callable_name)
     {

--- a/ext-src/php_swoole.h
+++ b/ext-src/php_swoole.h
@@ -912,7 +912,7 @@ static sw_inline zend_bool sw_zend_is_callable_at_frame(zval *zcallable, zval *z
 {
     zend_string *name;
     zend_bool ret;
-#if PHP_VERSION_ID < 80000
+#if PHP_VERSION_ID < 80001
     ret = zend_is_callable_ex(zcallable, zobject ? Z_OBJ_P(zobject) : NULL, check_flags, &name, fci_cache, error);
 #else
     ret = zend_is_callable_at_frame(zcallable, zobject ? Z_OBJ_P(zobject) : NULL, frame, check_flags, fci_cache, error);

